src/audio/mixer.c

```diff
@@ -560,6 +562,22 @@ static int mixout_params(struct comp_dev *dev,
 	params->buffer_fmt = md->base_cfg.audio_fmt.interleaving_style;
 	params->buffer.size = md->base_cfg.ibs;
 
+	/* update each sink format */
```

Comment:
What are we updating mixin from IPC hw params , topology or manifest ? Can you say in the comment.

Reply:
sure.  For IPC4,   module is initialized by base_cfg which is built by kernel driver according to runtime hw_params and topology.